js/game.js

```diff
@@ -0,0 +1,70 @@
+/** @type {HTMLCanvasElement} */
+
+class Game{
+    constructor(ctx, width, height, player){
+        this.ctx = ctx
+        this.width = width
+        this.height = height
+        this.player = player
+        this.intervalId = null
+        this.frames = 0
+        this.enemies = []
+    }
+
+    start(){
+        this.intervalId = setInterval(this.upDate, 1000/60) 
+    }
+
+     upDate = () => {
+        this.frames++
+        this.clear()
+        this.player.newPosition()
+        this.player.draw()
+        this.updateEnemies()
+        this.checkGameOver()
+    }
+
+    stop(){
+        clearInterval(this.intervalId)
+    }
+
+    clear(){
+        this.ctx.clearRect(0, 0, this.width, this.height)
+    }
+
+    updateEnemies(){
+        for(let i = 0; i < this.enemies.length; i++){
+            this.enemies[i].y += 4
+            this.enemies[i].draw()
+        }
+
+        if(this.frames % 60 === 0){
+            let randomSize = Math.floor(Math.random() * 150 - 20) + 20
+            let randomX = Math.floor(Math.random() * this.width - randomSize) + randomSize
+            this.enemies.push(new Enemy(randomX, 0, randomSize, randomSize, 'green',this.ctx));
```

Comment:
To make the obstacles horizontal you just need to change the width and the height right here where you are creating them:

```
(...)

if (this.frames % 60 === 0) {

      // Change the min size (50) and max size (250) values 
      let randomSize = Math.floor(Math.random() * 250 - 50) + 100;

      let randomX =
        Math.floor(Math.random() * this.width - randomSize) + randomSize;
      
      // Give a fixed values for the height of the obstacles (30)
      this.enemies.push(
        new Enemy(randomX, 0, randomSize, 30, "green", this.ctx)
      );
    }

(...)
```